ocaml/xapi/xapi_pif.ml

```diff
@@ -485,15 +485,30 @@ let scan ~__context ~host =
 			(List.map fst t.device_to_mac_table)
 			(List.map snd t.pif_to_device_table) in
 
+	let non_managed_devices =
+		if Sys.file_exists !Xapi_globs.non_managed_pifs then
+			try
+				let output, _ = Forkhelpers.execute_command_get_output !Xapi_globs.non_managed_pifs [] in
+				String.split_f String.isspace output
```

Comment:
Since OCaml 4.00.0 there's a String.trim function in the standard library which does the same thing.